### PR TITLE
Add codebase for the review

### PR DESCRIPTION
**Pull request by SeggyFault**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+}
+
+.code-preview-large {
+  display: grid;
+  grid-template-columns: 1fr;
+}
+
+pre code {
+  background-color: var(--code-bg-color);
+  font-size: 16px;
```
Comment (xJoshx) [inline css/styles.css:96]: Not sure but I think the default size for `body` is 16px already. Maybe makes more sense to define this in the `body` if you want to make it explicit so the whole page will inherit it

File: index.html [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+                    <p>
+                        The SCSS syntax uses the file extension .scss. With a few small exceptions, it’s a superset of
+                        CSS, which means essentially <strong>all valid CSS is valid SCSS as well.</strong> Because of
+                        its similarity to CSS, it’s the easiest syntax to get used to and the most popular.
+                    </p>
+                    <p>SCSS looks like this:</p>
+                    <div class="code-preview-large">
+                        <div class="code-preview-item-large">
+                            <pre>
+            <code>  <span class="mix">@mixin</span> <span class="property">button-base</span><span class="punctuation">() { </span>
```
Comment (xJoshx) [inline index.html:83]: I would suggest you to set up your code editor to automatically format your code so these things will be easier to read.

Screenshot 2021-08-04 at 09 06 08

File: index.html [hunk 3]
```diff
@@ -0,0 +1,365 @@
...
+  <span class="property">font-size</span><span class="punctuation">:</span> 12px
+  <span class="property">border-radius</span><span class="punctuation">:</span> 3px
+  <span class="property">border</span><span class="punctuation">:</span> 1px solid #e1e4e8
+                    </code>
+                  </pre>
+                        </div>
+                        <div class="code-preview-item">
+                            <span>CSS</span>
+                            <pre>
+                    <code>
```
Comment (xJoshx) [inline index.html:198]: Not sure which is the difference between these two tags, I would say with `pre` is enough 🤔

**Review by xJoshx — APPROVED** [review]
Left a couple of suggestions, looks good overall 👌

File: css/styles.css [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+  --mix-include-color: #000000;
+  /*warning*/
+  --warning-bg-color: #fcf7fa;
+  --warning-border-color: #eecdde;
+  /*fact*/
+  --fact-bg-color: #f6f9f9;
+  --fact-border-color: #c8dada;
+
+  /*PADDING*/
+  --block-padding: 30px;
```
[the remark this answers is not in the document]
Comment (SeggyFault) [reply css/styles.css]: I've added a spacing system similar to the one you've sent me

File: css/styles.css (hunk 1 above)
Comment (SeggyFault) [reply css/styles.css]: When I commented `font-size`, it got smaller, so I've added a `font-size` to `body` and `inherit` to `pre`

File: index.html (hunk 2 above)
Comment (SeggyFault) [reply index.html:83]: I can't format it since `pre` is sensible to the amount of spacing :confused:

File: css/styles.css (hunk 1 above)
Comment (xJoshx) [reply css/styles.css]: I'm a bit late to the party but I'll explain you why that happened: `body` by default has `16px` of font size, and `pre` has default font size much smaller. So, this one was overriding the `body` font size one. What you did is totally correct 👌

File: index.html (hunk 2 above)
Comment (xJoshx) [reply index.html:83]: True 😅 . Still, you will take advantage of formatting the `html` as it will be much readable for us, humans 🤖

**Pull request merged**